### PR TITLE
Add operator to connection metrics

### PR DESCRIPTION
**Pull request by w84thesun**
# Description

Return structured errors. Use those errors to enrich metrics and telemetry.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/handlers/common/update.go [hunk 1]
```diff
@@ -474,6 +474,12 @@ func HasSupportedUpdateModifiers(update *types.Document) (bool, error) {
 			fallthrough
 		case "$pop":
 			updateModifier = true
+		case "$mul", "$rename":
+			return false, NewErrorMsg(
+				ErrNotImplemented,
+				fmt.Sprintf("update operator %s is not implemented", updateOp),
+				NewErrOption("NotImplemented", updateOp),
```
[the remark this answers is not in the document]
Comment (w84thesun) [reply internal/handlers/common/update.go:481]: Yes, you're right. It should be called "operator". To completely eliminate that identifier we can introduce a set of Option functions but I'm not sure if that's a good choice.

[the remark this answers is not in the document]
Comment (AlekSi) [reply internal/handlers/common/update.go:481]: Let's keep it simple for now: no functions, no multiple options, just one operator

File: internal/clientconn/connmetrics/conn_metrics.go [hunk 2]
```diff
@@ -143,6 +143,8 @@ func (cm *ConnMetrics) GetResponses() map[string]CommandMetrics {
 				result = label.GetValue()
 			case "stage":
 				stage = must.NotFail(strconv.Atoi(label.GetValue()))
+			case "operator":
```
Comment (w84thesun) [inline internal/clientconn/connmetrics/conn_metrics.go:146]: @AlekSi should we do something with that label?

**Review by chilagrow — APPROVED** [review]
Looks good, this will allows us to access operator information for the metrics :muscle:

**Comment by AlekSi** [comment]
Moved into #1327.

**Pull request closed without merging**